optexp/datasets/wikitext103.py

```diff
@@ -0,0 +1,172 @@
+import os
+import shutil
+from pathlib import Path
+
+import pyarrow.parquet
+import requests
+import torch
+from attrs import frozen
+from torch.utils.data import DataLoader
+
+from optexp.config import Config
+from optexp.datasets import Dataset
+from optexp.datasets.dataset import TrVa, HasClassCounts, Downloadable
+from optexp.datasets.tokenizers import Tokenizer, BPETokenizer
+
+
+#  TODO <unk> considered harmful
```

Comment:
Add a `raw: bool = True` parameter to use the raw data instead of the already tokenized with `<unk>`?

Reply:
done